tests/test_pane.py

```diff
@@ -66,10 +67,13 @@ def test_set_width(session: Session) -> None:
 
 
 def test_capture_pane(session: Session) -> None:
+    env = shutil.which("env")
+    assert env is not None, "Cannot find usable `env` in PATH."
+
     session.new_window(
         attach=True,
         window_name="capture_pane",
-        window_shell='env -i PS1="$ " /usr/bin/env bash --norc --noprofile',
+        window_shell=f"{env} PS1='$ ' bash --norc --noprofile",
```

Comment:
But I was also saying over here that `bash` may not exist 😆  I will need to look at that later.